src/sentry/tasks/commit_context.py

```diff
@@ -83,6 +83,21 @@ def queue_comment_task_if_needed(
             )
         return
```

Comment:
you could have multiple open PRs with a single commit, but i think it'd be quite noisy if we commented on all of them, so not tracking the metric here